docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1468,6 +1527,62 @@ using Teku to sign blocks and attestations always uses its built-in slashing pro
 
 Timeout in milliseconds for requests to the external signer. Default is 1000.
 
+### validators-external-signer-truststore
+
+=== "Syntax"
+
+    ```bash
+    --validators-external-signer-truststore=<FILE>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --validators-external-signer-truststore=websigner_truststore.p12
+    ```
+
+=== "Environment Variable"
+
+    ```bash
+    TEKU_VALIDATORS_EXTERNAL_TRUSTSTORE=websigner_truststore.p12
+    ```
+
+=== "Configuration File"
+
+    ```bash
+    validators-external-signer-truststore: "websigner_truststore.p12"
+    ```
+
+PKCS12 or JKS keystore used to trust external signer's self-signed certificate
```

Comment:
suggestion similar to: `PKCS12 or JKS keystore used to trust external signer's self-signed certificate or CA certificate which signs external signer's certificate`

Reply:
Done